manifests/RoniLehto/Lmath/1.6.yaml

```
@@ -0,0 +1,20 @@
+Id: RoniLehto.LMath
+Publisher: Roni Lehto
+Name: L'Math
+Version: 1.6
+License: (c) Roni Lehto 2020
+InstallerType: INNO
+LicenseUrl: https://www.lehtodigital.fi/lmath/?p=license
+AppMoniker: LMath
+Tags: math, latex, editor
+Description: L'Math is an Electron powered text editor that is mainly used by Finnish students preparing for their digital matriculation exams. L'Math has the same LaTeX editor built-in that is found on the Digabi platform.
+Homepage: https://www.lehtodigital.fi/lmath/
+Installers:
+  - Arch: x86
+    InstallerType: EXE
+    Url: https://www.lehtodigital.fi/lmath/proxy.php?download=win32
```

```suggestion
    Url: https://github.com/lehtoroni/lmath-issues/releases/download/r1.6/LMath_Windows_r1.6.exe
```
`https://www.lehtodigital.fi/lmath/proxy.php?download=win32` redirects to `https://github.com/lehtoroni/lmath-issues/releases/download/r1.6/LMath_Windows_r1.6.exe`